committee-steering/governance/sig-governance.md

```diff
@@ -140,6 +141,17 @@ curation from other SIG participants
   - Number: 2-3
   - Membership tracked in [sigs.yaml]
 
+### Program Manager
+
+- *Optional Role*: SIG Program Managers
+  -  Coordinate the success of their SIGs’ delivery workflow and related practices
+  -  Coordinate cross-SIG coordination
+  -  Set up, launch, and continuously optimize processes
+  -  Drive prioritization, delegation, communication, risk mitigation efforts
```

Comment:
I have some concerns that this phrasing, i.e. "drive prioritization", may have a risk when applied broadly across all SIGs.

It should clarify that if a role like this was adopted, the individuals in this role do not control resource allocation within a SIG.  At some level, driving prioritization requires mutual consensus between the individual proposing a change and the individual reviewing it both in time and support and I worry this language could be easily confused as delegating greater powers than intended.